sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/FormTrainingClientTestBase.java

```diff
@@ -306,15 +277,18 @@ String getApiKey() {
     String getEndpoint() {
         return interceptorManager.isPlaybackMode()
             ? "https://localhost:8080"
-            : Configuration.getGlobalConfiguration().get("AZURE_FORM_RECOGNIZER_ENDPOINT");
+            : Configuration.getGlobalConfiguration().get(AZURE_FORM_RECOGNIZER_ENDPOINT);
     }
 
-    private String createStorageAndGenerateSas(String folderPath) {
-        if (interceptorManager.isPlaybackMode()) {
-            return "https://isPlaybackmode";
-        } else {
-            return Configuration.getGlobalConfiguration().get(FORM_RECOGNIZER_TRAINING_BLOB_CONTAINER_SAS_URL);
+    private String getTrainingSasUri() {
+        return interceptorManager.isPlaybackMode()
+            ? "https://isPlaybackmode"
+            : Configuration.getGlobalConfiguration().get(FORM_RECOGNIZER_TRAINING_BLOB_CONTAINER_SAS_URL);
+    }
 
-        }
+    private String getTestingSasUri() {
+        return interceptorManager.isPlaybackMode()
+            ? "https://isPlaybackmode"
+            : Configuration.getGlobalConfiguration().get(FORM_RECOGNIZER_TESTING_BLOB_CONTAINER_SAS_URL);
```

Comment:
These two methods can simplify to be one generic method.
```
private String getSasUri(String sasUrl) {}

``

Reply:
they are returning different configurations